### DISCUSSION TITLE
Access to the html file

### DISCUSSION THREAD

**Discussion by Skavengers** [oobabooga/text-generation-webui#3337] [Q&A]
Hello
Stupid ask but I don’t see the html file, and I need it. Because I want to make a chatbot game with canvas(pixijs).
So I would be glad if someone could show me where. I also apologize for any english misstakes I certainly make in this post

**Comment by missionfloyd** [comment]
There isn't a static HTML file. The interface is made with [gradio](https://www.gradio.app/) and is defined in [server.py](https://github.com/oobabooga/text-generation-webui/blob/main/server.py)
